main/lsp/wrapper.cc

```diff
@@ -106,4 +107,8 @@ int LSPWrapper::getTypecheckCount() {
     return lspLoop->getTypecheckCount();
 }
 
+const LSPConfiguration &LSPWrapper::config() const {
```

Comment:
A _lot_ of test code manually plumbed through root path and root URI information that we already centralized in `LSPConfiguration`! So, I decided to add this accessor to the wrapper so that test code can query the config object directly for this info.